Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># THIS IS GOOD SH*T LOL...\r\nrdfff\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	
+++ b/README.md	
@@ -1,4 +1,4 @@
 # THIS IS GOOD SH*T LOL...
-rdfff
+gttt
 
 
